bioio/array_like_reader.py

```diff
@@ -110,13 +110,29 @@ class ArrayLikeReader(Reader):
     def _is_supported_image(  # type: ignore
         image: Union[List[MetaArrayLike], MetaArrayLike], *args: Any, **kwargs: Any
     ) -> bool:
+        errors = []
```

Comment:
A list of strings called `errors` is pretty confusing to me. Why not use real errors?